### ISSUE TITLE
NN regressors?

### ISSUE THREAD

**Issue by fcoppey** [EpistasisLab/tpot#1081]
Is there / will be any possibility to use pytorch NN for regression? 
I find these only as classifiers now.

Thanks a lot for the amazing work

**Comment by weixuanfu** [comment]
I think @JDRomano2 will work on it.

**Comment by JDRomano2** [comment]
Yes, this is a planned addition for the near future. Since this started as a proof-of-concept, we worked on NN classifiers only to make sure those would function as intended.

I'll update this thread as we add support for NN regressors.
